src/helpers/enrich-dataset.ts

```diff
@@ -55,12 +54,35 @@ export function enrichDataset(dataset: HubDataset, hubsite: HubSite) {
         additionalFields.accessUrlWMS = ogcUrl(dataset.url, 'WMS');
     }
 
-    return {
+    return hubDatasetToFeature({
         ...dataset,
         ...additionalFields
-    };
+    });
 };
 
+function hubDatasetToFeature(hubDataset: HubDataset) {
+    const { type, rings } = hubDataset?.boundary?.geometry ?? {};
+
+    return {
+        type: 'Feature',
+        geometry: {
+            type: type && capitalize(type),
+            coordinates: rings
+        },
+        properties: _.omit(hubDataset, ['boundary'])
```

Comment:
I've read that `omit` is inefficient.  Check this post for alternatives: https://github.com/FormidableLabs/victory/issues/956

Reply:
Used babel transpiler method to remove key from an object. Considered to be the fastest.